src/tasks/BagelAITools.py

```
@@ -0,0 +1,1157 @@
+        # 动态从 /v1/models 探测当前模型名
+        # ========================================== 
+        model_name = "local-model"  # 缺省兜底值
+        preferred_model = self.config.get(self.CONF_MODEL_NAME, "qwen/qwen3-vl-4b") # 指定主导模型
+        models_url = f"{base_url}/v1/models"
+        models_response = requests.get(models_url, headers=headers, timeout=3)
+        if models_response.status_code == 200:
+            models_data = models_response.json()
+            if "data" in models_data and len(models_data["data"]) > 0:
+                # 提取出当前后端所有可用的模型 ID 列表
+                available_model_ids = [m["id"] for m in models_data["data"]]
+                # 策略 1：检查我们最爱的 qwen/qwen3-vl-4b 在不在里面
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)